openfecwebapp/models/committees.py

```diff
@@ -0,0 +1,29 @@
+from marshmallow import Schema, fields
+from flask import url_for
+
+
+class CommitteeSchema(Schema):
```

Comment:
Again, I like this way better than the code it's replacing, but do we even need a schema here? We're just passing most fields through unaltered, and IMO the fields that we do change aren't essential. For example, why not pass through `designation` exactly as we get it from the API and use code like `{% if committee.designation == 'P' %}` in the templates?